Formula/r/redis.rb

```diff
@@ -9,8 +9,7 @@ class Redis < Formula
   head "https://github.com/redis/redis.git", branch: "unstable"
 
   livecheck do
-    url "https://download.redis.io/releases/"
-    regex(/href=.*?redis[._-]v?(\d+(?:\.\d+)+)\.t/i)
+    skip "7.2.4+ is no longer OSS"
```

Comment:
```suggestion
    skip "Versions after 7.2.4 use non-open source license"
```

We may want to reword this for the sake of clarity, as it may read as "7.2.4 and above" instead of "after 7.2.4".